strategy/protobuf/internal/json.go

```diff
@@ -0,0 +1,39 @@
+package internal
+
+import (
+	"encoding/json"
+
+	"github.com/mailru/easyjson"
+	"github.com/mailru/easyjson/jwriter"
+)
+
+// MarshalJSON returns the JSON encoding of v.
+// This is done using `easyjson.Marshaler`, `json.Marshaler` or `json.Marshal`
+func MarshalJSON(v interface{}) ([]byte, error) {
```

Comment:
Why do we still have this in the `protobuf` strategy?

Reply:
Used for metadata marshaling in strategy/protobuf/sql/postgres/single_stream_strategy.go:83.

Reply:
Oh :man_facepalming: right!